README.md

```diff
@@ -89,9 +89,14 @@ Per default [Sqlite3](https://sqlite.org/) is used as database for simple deploy
 Per default no authentication is needed. To protect api, integrate it with your [IAM](https://en.wikipedia.org/wiki/Identity_management) supporting Open ID Connect. If not availbale you might consider using [Keycloak](https://www.keycloak.org/).
 
 * `REQUIRE_AUTHENTICATION`: Force authentication to be required (default: False)
-* `GROUP_ACCESS_ONLY`: Force visibility to templates of group only. See also `OIDC_GROUPS_CLAIM`. (default: False)
+* `GROUP_ACCESS_ONLY`: Force visibility to templates of group only. See also `OIDC_GROUPS_CLAIM` or `OIDC_GROUPS_API` (default: False)
 * `OIDC_USERINFO_ENDPOINT`: Url of userinfo endpoint as [described](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo)
+* `OIDC_VERIFY_SSL`: Verify ssl certificate of oidc userinfo endpoint (default: True)
 * `OIDC_GROUPS_CLAIM`: Name of claim to be used to define group membership (default: document_merge_service_groups)
+* `OIDC_GROUPS_API`: In case authorization is done in a different service than your OpenID Connect provider you may apply an json api endpoint which is called after authentication. Use `{sub}` as placeholder for username. Replace `sub` with any claim name.
```

Comment:
```suggestion
* `OIDC_GROUPS_API`: In case authorization is done in a different service than your OpenID Connect provider you may specify an API endpoint return JSON which is called after authentication. Use `{sub}` as placeholder for username. Replace `sub` with any claim name.
```

Reply:
Maybe also add an example for a valid setting using `{sub}`.

Reply:
Adjusted as suggested and added example url